setup.py

```diff
@@ -192,9 +169,6 @@
     "unidic_lite>=1.0.7",
     "urllib3<2.0.0",
     "uvicorn",
-    "pytest-rich",
```

Comment:
no references to `pytest-rich` (library name) nor `--rich` (the flag it enables) in our library

Reply:
maybe it's there in case someone want to use this pytest feature (that's why it is not seen in our library)